sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/AzureCosmosPropertiesTest.java

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.cosmos;
+
+import com.azure.core.util.CoreUtils;
+import com.azure.cosmos.implementation.HttpConstants;
+import org.testng.annotations.Test;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class AzureCosmosPropertiesTest {
+
+    @Test(groups = "unit")
+    public void verifyAzureCosmosProperties() {
+        Map<String, String> properties =
+            CoreUtils.getProperties(HttpConstants.Versions.AZURE_COSMOS_PROPERTIES_FILE_NAME);
+        assertThat(properties).isNotNull();
+        assertThat(properties).isNotEmpty();
+        assertThat(properties.get("version")).isNotNull();
+        assertThat(properties.get("name")).isNotNull();
+    }
+
+    @Test(groups = "unit")
+    public void verifyProjectVersion() {
+        assertThat(HttpConstants.Versions.SDK_VERSION).isNotNull();
+        String pomFileVersion = getVersionFromPomFile();
+        assertThat(HttpConstants.Versions.SDK_VERSION).isEqualTo(pomFileVersion);
+    }
+
+    private String getVersionFromPomFile() {
+        String fileName = "pom.xml";
+        String versionStartTag = "<version>";
+        String versionEndTag = "</version>";
+        File file = new File(fileName);
+        try {
```

Comment:
ideally you can use xml parser to get the path of the filed of interest
anyway, not a big deal for a test and may not worth the time.

Thank you.

Reply:
Yeah, didn't want to add extra dependency :) 
but yes, I agree. We could use that.